src/dipdup/env.py

```diff
@@ -54,6 +54,8 @@ def get_package_path(package: str) -> Path:
         return Path.cwd()
 
     # NOTE: Detect existing package in current environment
+    if Path.cwd().name == package:
```

Comment:
Let's perform this check after regular importing failed. Also, please add an entry to the changelog and run `make docs`.